http/oauth2/oauth2.go

```diff
@@ -84,7 +84,12 @@ func introspectRequest(r *http.Request, w http.ResponseWriter, tokenIntro TokenI
 			log.Req(r).Info().Msg(err.Error())
 			http.Error(w, err.Error(), http.StatusUnauthorized)
 			return nil, false
```

Comment:
`case ErrInvalidToken` is identical to the `default` case, so you can omit it. Or explicitly add `fallthrough`